Build/Projects/MonoGame.Framework.definition

```diff
@@ -44,6 +43,7 @@
     <CustomDefinitions>
       <Platform Name="Android">TRACE;ANDROID;GLES;OPENGL;OPENAL</Platform>
       <Platform Name="iOS">IOS;GLES;OPENGL;OPENAL</Platform>
+      <Platform Name="tvOS">IOS;__TVOS__;GLES;OPENGL;OPENAL</Platform>
```

Comment:
To be fair the Xamarin Platform(s) already define these constants like this 

__TVOS__
__IOS__
__ANDROID__

we should probably migrate over to using those if possible, but didn't want to do that all in this PR. It was my suggestion to stick with __TVOS__ we can change it if needed.